testproject/Assets/Scripts/ConnectionModeScript.cs

```diff
@@ -72,25 +69,38 @@ private IEnumerator WaitForNetworkManager()
         yield return null;
     }
 
+    /// <summary>
+    /// Check whether we are even using UnityTransport and
+    /// if so whether it is using the RelayUnityTransport
+    /// </summary>
+    private bool HasRelaySupport()
+    {
+        var unityTransport = NetworkManager.Singleton.GetComponent<UnityTransport>();
+        if (unityTransport != null && unityTransport.Protocol == UnityTransport.ProtocolType.RelayUnityTransport)
```

Comment:
If the relay SDK is not present then we should just always return false hence the defines.

Reply:
Ok, that makes sense.  I think for our testproject we will always have it available from this point forward?  If that is not the case then I can put the defines back where they were.